### ISSUE TITLE
ITIS references missing

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#387]
See https://github.com/CatalogueOfLife/data/issues/386

For example, compare:
_Gastrophryne carolinensis_ (Holbrook, 1835) ([TSN 173467](https://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=173467#null))
vs:
https://www.catalogueoflife.org/data/taxon/3FDVS
and https://data.catalogueoflife.org/dataset/2144/taxon/173467

Also there is just one reference for ASW: https://data.catalogueoflife.org/dataset/2144/verbatim?q=Amphibian%20Species%20of%20the%20World

**Comment by DaveNicolson** [comment]
That citation for ASW from ITIS is for a specific _published_ version, not for the electronic version we are discussing. ITIS cites the current ASW website as an "other source" rather than as a "publication" (or an "expert", the 3rd type of reference in ITIS). The link to show all the names in ITIS that cite the (2019) version of ASW used to update ITIS is here, but probably won't load due to the thousands of names citing for it:
https://www.itis.gov/servlet/SingleRpt/RefRpt?search_type=source&search_id=source_id&search_id_value=1195

**Comment by mdoering** [comment]
Ok, If I understand "Other Sources" correctly then this is not a source used for compilation, but a source known to be of interest that treats the taxon also well. A "see also" reference for further research? In that case I would probably not include them in the lists in col:Taxon.referenceID. 

@gdower there still is an issue with the publications included as there are just 2 (6,3538) in ColDP while ITIS shows many more.

**Comment by DaveNicolson** [comment]
No, ITIS uses 3 kinds of references: actual publications, experts, and other sources. The latter if for almost anything, but most commonly is website or database. **There is no flag for what was used for compilation**, unfortunately. But in this case, the ASW "other source" (SRC 1195 in this case) is THE MAIN SOURCE WE USED. All three reference types are cited with the same mechanism in ITIS, the reference_links_table, which uses source type/prefix (PUB, EXP, or SRC) and reference_id as the key to the 3 reference tables (publications, experts, other_sources). There is a flag for 'original description' in the reference_links table that can be used for PUB citations for any given TSN. But in this case, there is nothing inherent to any of the reference links to tell you "this was the main source for the taxonomic data".... We had a similar quandry when working with the 'spice wrapper' way back when. There is no perfect solution for that kind of flag currently.

**Comment by mdoering** [comment]
Would you then think it is best to include all references for [col:Taxon.referenceID](https://github.com/CatalogueOfLife/coldp/blob/master/README.md#referenceid-3):

> A comma concatenated list of reference IDs supporting the taxonomic concept that has been reviewed by the scrutinizer.

**Comment by DaveNicolson** [comment]
That is probably the best we can do... if it works... Most TSNs in ITIS link to a small number of references, but some have a number.

The example I used above somehow ended up with a ~duplicate PUB (the PUB dated 2000), so it is not the perfect example, but so it goes...
